### PR TITLE
Monitoring 6.0 supervisord FINAL

### PR DESCRIPTION
**Pull request by kevinmtrowbridge**
@jleealpine's & my work, consolidated into this branch, which we may be able to merge to master after a period of QA.
* basic supervisord installer
* basic supervisord scripts
* cleanup and reorganization of packaging directory

### CODE REVIEW

**Comment by jleealpine** [comment]
OK, I changed up the "disabled" conf files to be "template" conf files contained in their own folder and simply do a copy of that file to the main "packaging" folder when enabling that conf file, and deleting it from "packaging" if disabling that conf file.

Also, added passing in a "prepare" parameter to alpine_control.sh.  For now, it will do nothing and still attempt to start Alpine, the Alpine side changes are needed to do this properly.

**Comment by jleealpine** [comment]
Also, here is the pull request I opened up for the supervisord changes on the Alpine side.  It's just a matter of making sure that start_alpine.sh doesnt try to start Alpine on its own as well, we will pass in the "prepare" parameter to get it to do all the other stuff.

https://github.com/alpinedatalabs/adl/pull/1288

**Comment by kevinmtrowbridge** [comment]
I will go through this with a "fine-toothed comb" in preparation for merging it, pointing out details that we may be missing.

File: components/admin/app/views/admin/dashboard/index.html.erb [hunk 1]
```diff
@@ -44,6 +44,10 @@
         <ul class="list-unstyled">
 
           <li>
+            <%= link_to "Process Control", "http://#{request.host}:9001/", :target => "_blank" %>
```
Comment (kevinmtrowbridge) [inline components/admin/app/views/admin/dashboard/index.html.erb]: Anyone will be able to see the supervisord control screen if they visit `http://chorusurl:9090` ...

There's the possibility of passing passwords to make it a little more secure:
http://supervisord.org/configuration.html#supervisorctl-section-values

**Pull request merged**